### PR TITLE
non-split-shopping-list-endpoint

### PR DESCRIPTION
**Pull request by shubham-khandelwal2-metacube**
## PR Description
https://spryker.atlassian.net/browse/CC-16844

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    ...    AND    Response status code should be:    204
+    ...    AND    Response reason should be:    No Content
+    ...    AND    I send a DELETE request:    /shopping-lists/${shoppingListId2}
+    ...    AND    Response status code should be:    204
+    ...    AND    Response reason should be:    No Content
+
+Get_shopping_lists_info_with_non_zero_quantity_of_number_of_items
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user.email}
+    ...    AND    I set Headers:    Content-Type=${default_header_content_type}    Authorization=${token}  
+    I send a GET request:    /shopping-lists
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot:156]: Don't we need to create a shopping list with items here during setup?

File: tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    And Response status code should be:    200
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+    And Response body parameter should NOT be:    [data][0][attributes][numberOfItems]    "None"
+
+Get_shopping_lists_info_for_user_with_zero_quantity_of_number_of_shopping_lists
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_fourth_user.email}
+    ...    AND    I set Headers:    Content-Type=${default_header_content_type}    Authorization=${token}
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot:163]: Don't we need to create a shopping list here during setup?

File: tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    And Response status code should be:    200
+    And Save value to a variable:    [data][attributes][updatedAt]    updatedAt
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+    And Response body parameter should be:    [data][attributes][name]    ${shopping_list_name}${random}!@#$%^&*()-_
+    And Response body parameter should be:    [data][attributes][updatedAt]    ${updatedAt}
+    [Teardown]    Run Keywords    I send a DELETE request:    /shopping-lists/${shoppingListId}
+    ...    AND    Response status code should be:    204
+    ...    AND    Response reason should be:    No Content
+
+# b2b2 - There is a bug CC-16543
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot:73]: Is this bug relevant for suite-non split? Comment says that it is only for b2b

File: tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    And Response should contain the array smaller than a certain size:    [data][relationships]    ${numberOfItems}+1
+    And Response include should contain certain entity type:    concrete-products
+    And Response include should contain certain entity type:    shopping-list-items
+    And Response include element has self link:   concrete-products
+    And Response include element has self link:   shopping-list-items
+    [Teardown]    Run Keywords    I send a DELETE request:    /shopping-lists/${shoppingListId}
+    ...    AND    Response status code should be:    204
+    ...    AND    Response reason should be:    No Content
+
+# b2b - There is a bug CC-16541
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot:197]: Is this bug relevant for suite-non split? Comment says that it is only for b2b

File: tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot (hunk 1 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot:156]: FIXED. Added create /shopping-lists in setup.

File: tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot (hunk 4 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/shopping_list_endpoints/shopping_lists/positive.robot:197]: DONE

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Pull request merged**